_posts/2018-08-10-summer-code-sprint-version-1_9.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: GeoBlacklight 2018 Summer Sprint Wrap-up
+date: 2018-08-10
```

Comment:
I suspect that this also needs a time.  Other posts use `12:00:00` (e. g. date: 2018-08-10 12:00:00): https://raw.githubusercontent.com/geoblacklight/geoblacklight.github.io/master/_posts/2015-04-10-deploying-geoblacklight.markdown